rotkehlchen/tests/exchanges/test_okx.py

```diff
@@ -0,0 +1,802 @@
+         "from":"",
+         "to":"0x388c818ca8b9251b393131c08a736a67ccb19297",
+         "state":"2",
+         "nonTradableAsset":false,
+         "ts":"1670953159000",
+         "wdId":"66666666",
+         "feeCcy":"USDT"
+      }
+   ],
+   "msg":""
+}
+            """
+        else:
+            data = ""
+        return MockResponse(200, data)
+
+    with patch.object(
+            mock_okx.session,
+            'request',
+            side_effect=mock_okx_deposits_withdrawals,
+    ):
+        asset_movements = mock_okx.query_online_deposits_withdrawals(
+            Timestamp(1609103082),
+            Timestamp(1672175105),
+        )
+
+    expected_asset_movements = [
+        AssetMovement(
+            location=Location.OKX,
+            category=AssetMovementCategory.DEPOSIT,
+            timestamp=Timestamp(1669963555),
+            address='0xaab27b150451726ec7738aa1d0a94505c8729bd1',
+            transaction_id='0xfd12f8850218dc9d1d706c2dbd6c38f495988109c220bf8833255697b85c92db',
+            asset=A_USDT,
+            amount=FVal(2500.180327),
+            fee_asset=A_USDT,
+            fee=Fee(FVal(0)),
+            link='0xfd12f8850218dc9d1d706c2dbd6c38f495988109c220bf8833255697b85c92db',
+        ),
+        AssetMovement(
+            location=Location.OKX,
+            category=AssetMovementCategory.DEPOSIT,
+            timestamp=Timestamp(1669405596),
+            address='0xaab27b150451726ec7738aa1d0a94505c8729bd1',
+            transaction_id='0xcea993d53b2c1f79430a003fb8facb5cd6b83b6cb6a502b6233d83eb338ba8ba',
+            asset=A_USDC,
+            amount=FVal(990.795352),
+            fee_asset=A_USDC,
+            fee=Fee(FVal(0)),
+            link='0xcea993d53b2c1f79430a003fb8facb5cd6b83b6cb6a502b6233d83eb338ba8ba',
+        ),
+        AssetMovement(
+            location=Location.OKX,
+            category=AssetMovementCategory.WITHDRAWAL,
+            timestamp=Timestamp(1671542569),
+            address='9ZLfHwxzgbZi3eiK43duZVJ2nXft3mtkRMjs9YD5Yds2',
+            transaction_id='46tgp3RHNuQqQrHbms1NtPFkRRwsabCajvEUPX'
+                           'BryVuH6qJmQysn1V9VhTYBEJmVQq8s8fbfv4WFW3oj2LtwRzyU',
+            asset=A_SOL,
+            amount=FVal(49.86051649),
+            fee_asset=A_SOL,
+            fee=Fee(FVal(0.008)),
+            link='46tgp3RHNuQqQrHbms1NtPFkRRwsabCajvEUPXBryVuH6qJm'
+                 'Qysn1V9VhTYBEJmVQq8s8fbfv4WFW3oj2LtwRzyU',
+        ),
+        AssetMovement(
+            location=Location.OKX,
+            category=AssetMovementCategory.WITHDRAWAL,
+            timestamp=Timestamp(1670953159),
+            address='0x388c818ca8b9251b393131c08a736a67ccb19297',
+            transaction_id='0x9444b018c33c5adb58ee171bc18e61c56078495e37ae88833007a46c02b4552f',
+            asset=A_USDT,
+            amount=FVal(421.169831),
+            fee_asset=A_USDT,
+            fee=Fee(FVal(0.1)),
+            link='0x9444b018c33c5adb58ee171bc18e61c56078495e37ae88833007a46c02b4552f'),
+    ]
```

Comment:
Should those be changed?

Reply:
I have anonymized (by using random values) hopefully all instances of identifiable data - addresses, transaction hashes and unique identifiers for trades, deposits & withdrawals